prompts/power-pages/rental-property-listing/README.md

```diff
@@ -0,0 +1,31 @@
+# YOUR PROMPT NAME 
+
+Rental Property Listing
+
+## Description
+
+This prompt helps with creating a customer-facing Power Pages site to showcase available rental properties and enable inquiries.
+
+## Prompt
+
+Create a Power Pages site for customers interested in rental properties. The site should display available properties with details like address, type, status, and images. Customers should be able to inquire about a property through a contact form.
+
+### Supported Language(s)
+
+[English](./en-us/prompt.md)
+
+## Authors
+
+Solution|Author(s)
+--------|---------
+Rental Property Listing | [Markus Franz](https://www.github.com/mmbr1606)
+## Minimal Path to Awesome
+
+* Copy the prompt
+* Paste prompt into the tool of your choice (AIBuilder Azure Open AI, Power Apps Copilot, etc)
+
+## Disclaimer
+
+**THIS CODE IS PROVIDED *AS IS* WITHOUT WARRANTY OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING ANY IMPLIED WARRANTIES OF FITNESS FOR A PARTICULAR PURPOSE, MERCHANTABILITY, OR NON-INFRINGEMENT.**
+
+<img src="https://m365-visitor-stats.azurewebsites.net/powerplatform-prompts/samples/ai-builder/sample" aria-hidden="true" />
```

Comment:
Please change to `<img src="https://m365-visitor-stats.azurewebsites.net/powerplatform-prompts/prompts/power-pages/rental-property-listing" aria-hidden="true" />`